### PR TITLE
[chore] Disabled chunks + dynamically load lottie animations

### PR DESCRIPTION
**Pull request by marcinciarka**
Before:
```
Route (pages)                                         Size     First Load JS
┌ ● /                                                 4.77 kB        3.42 MB
├   /_app                                             0 B            2.85 MB
[...]
+ First Load JS shared by all                         2.85 MB
  ├ chunks/main-ea6066b102e1c906.js                   106 kB
  ├ chunks/pages/_app-ab35137812f47f61.js             2.74 MB
  └ css/73a09d7ae5b15216.css                          325 B
```

After:
```
Route (pages)                                         Size     First Load JS
┌ ● /                                                 4.78 kB        2.77 MB
├   /_app                                             0 B            2.72 MB
[...]
+ First Load JS shared by all                         2.72 MB
  ├ chunks/1783-bb26c4f6fd3c15bb.js                   102 kB
  ├ chunks/6488-4e7f2e059e6eb75c.js                   1.85 MB
  ├ chunks/main-77eecde3d6c4caa5.js                   4.54 kB
  ├ chunks/pages/_app-e768a883ca8e5880.js             762 kB
  └ css/73a09d7ae5b15216.css                          325 B
  ```

### CODE REVIEW

**Review by piotrwitek — COMMENTED** [review]
Looks great

File: theme/animations/AddingStopLossAnimation.tsx
Comment (piotrwitek) [inline theme/animations/AddingStopLossAnimation.tsx:1]: using outdated file name convention, right now we should use `kebab-case`

**Review by piotrwitek — CHANGES_REQUESTED** [review]
Please apply to all changed files
Also please resolve sonar issues => 2 code smells

File: theme/animations/AddingStopLossAnimation.tsx
Comment (marcinciarka) [reply theme/animations/AddingStopLossAnimation.tsx:1]: This is now well scented.

**Review by jakubswierczek — APPROVED** [review]

**Review by piotrwitek — APPROVED** [review]

**Review by piotrkonowrocki — APPROVED** [review]

**Pull request merged**